packages/useful-types/test-d/types.test-d.ts

```diff
@@ -0,0 +1,20 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import {expectType, expectAssignable, expectNotAssignable} from 'tsd';
+
+import {ArrayElement} from '../build/ts/types';
```

Comment:
Rather than relying on build output are you able to test against `../src` instead?

Reply:
Replying to both ^ and `I don't understand why you're trying to jam this into jest,`

I like using the CLI since it works out of the box and if we have a nice way of running it then I'm all in! Thanks for setting up the orchestration!

Regrading some of the feedback, I didn't set out to use jest, just didn't want to / didn't know how to add another CI step while it was possible to do without. Also I think there is at least _some_ sense in running this check in the "test" phase since it is logically most similar to a unit test, as in setup, render, assert.